pyedflib/highlevel.py

```diff
@@ -327,19 +327,19 @@ def read_edf(edf_file, ch_nrs=None, ch_names=None, digital=False, verbose=False)
 
     with pyedflib.EdfReader(edf_file) as f:
         # see which channels we want to load
-        available_chs = [ch.upper() for ch in f.getSignalLabels()]
+        available_chs = f.getSignalLabels()
```

Comment:
why is this changed?

Reply:
It was to simplify, but I agree it is unrelated to the PR.